src/java.base/share/classes/sun/security/x509/SerialNumber.java

```diff
@@ -100,9 +101,8 @@ public SerialNumber(InputStream in) throws IOException {
     /**
      * Return the SerialNumber as user readable string.
      */
-    public String toString() {
-        return "SerialNumber: [" + Debug.toHexString(serialNum) + ']';
-    }
+    public String toString() { return "Serial number: " +
```

Comment:
Here I would keep "Serial number" as "SerialNumber" to maintain consistency with other X.509 `toString` methods which generally emit the name of the field using camel case format.

My only other comment is to whether it makes sense to move this into a utility method. I see `sun.security.util.Debug.toString` already does the same thing and uses HexFormat but in lower case. I think using lower case or upper case is not a big deal. So maybe just change all of these code instances to call `Debug.toString` to keep things consistent and reuse existing code.